src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.String.cs

```diff
@@ -12,12 +12,24 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
-        private static readonly System.Text.Encoding s_utf8Encoding = new System.Text.UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
+        // Implements major type 2,3 decoding per https://tools.ietf.org/html/rfc7049#section-2.1
 
-        // stores a reusable List allocation for keeping ranges in the buffer
+        // stores a reusable List allocation for storing indefinite length string chunk offsets
         private List<(int Offset, int Length)>? _indefiniteLengthStringRangeAllocation = null;
 
-        // Implements major type 2 decoding per https://tools.ietf.org/html/rfc7049#section-2.1
+        /// <summary>
+        ///   Reads the next data item as a byte string (major type 2).
+        ///   Both definite and indefinite-length strings are supported.
```

Comment:
Generally summaries are supposed to be one sentence.  This seems more like a remarks entry.